addons/resource.language.en_gb/resources/strings.po

```diff
@@ -22154,3 +22154,18 @@ msgctxt "#39119"
 msgid "Last modified"
 msgstr ""
 
+#.Display HW Scaling Filter Setting Toggle
+msgctxt "#39120"
+msgid "Display HW Scaling Filter"
```

Comment:
![hardware scaling filter](https://user-images.githubusercontent.com/531482/91664818-6b117b80-eaa6-11ea-8bb3-906d0ca34781.png)

HW is a technical abbreviation. And settings should be "Sentence case".

I would call this "Hardware scaling filter" or "Display hardware scaling filter"

Reply:
Yes, I will change it to "Display hardware scaling filter"